guides/source/security.md

```diff
@@ -32,21 +32,17 @@ In order to develop secure web applications you have to keep up to date on all l
 Sessions
 --------
 
-A good place to start looking at security is with sessions, which can be vulnerable to particular attacks.
+This chapter describes some particular attacks related to sessions, and security measures to protect your session data.
 
 ### What are Sessions?
 
-NOTE: _HTTP is a stateless protocol. Sessions make it stateful._
+INFO: Sessions enable the application to maintain user-specific state, while users interact with the application. Once the user has been authenticated, an active session enables the user keep using the application, without authenticating on each request.
```

Comment:
`an active session enables the user` _to_ `keep using the application` and also remove the comma before `without` (doesn't seem necessary)?

Thoughts?